Royalty/DefInjected/ResearchProjectDef/ResearchProjects_Implants.xml

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageData>
+  
+  <!-- EN: artificial metabolism -->
+  <ArtificialMetabolism.label>искусственный обмен веществ</ArtificialMetabolism.label>
+  <!-- EN: Craft bionics that process food more safely or efficiently than a biological stomach. -->
+  <ArtificialMetabolism.description>Создание бионических органов позволит обрабатывать пищу безопаснее и эффективнее, чем обычный желудок.</ArtificialMetabolism.description>
+  
+  <!-- EN: brain wiring -->
+  <BrainWiring.label>мозговая проводка</BrainWiring.label>
+  <!-- EN: Craft brain implants that induce or prevent blunt sensations like joy or pain. -->
+  <BrainWiring.description>Создание мозговых имплантов, которые вызывают или предотвращают примитивные ощущения, такие как радость или боль.</BrainWiring.description>
+  
+  <!-- EN: circadian influence -->
+  <CircadianInfluence.label>циркадное воздействие</CircadianInfluence.label>
+  <!-- EN: Craft brain implants that chemically manipulate the circadian rhythm, reducing or altering the need to sleep. -->
+  <CircadianInfluence.description>Создание мозговых имплантов, которые химически манипулируют циркадным ритмом, сокращая необходимость во сне.</CircadianInfluence.description>
```

Comment:
```suggestion
  <CircadianInfluence.description>Создание мозговых имплантов, которые химически управляют циркадным ритмом, сокращая потребность во сне.</CircadianInfluence.description>
```

Reply:
Fixed in 2f8b2218666114cd0179c623af96a3c7184bb93d